checkST2110.js

```diff
@@ -454,8 +465,32 @@ const test_10_62_1 = (sdp, params) => {
   return errors;
 };
 
-// Function to check that rtpmap is present and has passed in type and clockRate.  
-// An optional specification string can be used to be included in errors 
+const tsmodePermitted2022 = ['SAMP', 'NEW', 'PRES'];
+
+// ST 2110-10:2022 Section 8.7 Test 1 - RTP Timestamp Mode and Delay, TSMODE and TSDELAY parameters are an acceptable value.
+const test_10_2022_87_1 = (sdp, params) => {
+  let [mtParams, errors] = extractMTParams(sdp, { checkDups: true });
+  for (let stream of mtParams) {
+    let keys = Object.keys(stream);
+    if (keys.indexOf('TSMODE') >= 0) {
+      if (tsmodePermitted2022.indexOf(stream.TSMODE) < 0) {
+        errors.push(new Error(`Line ${stream._line}: For stream ${stream._streamNumber}, format parameter 'TSMODE' is not set to the required value as per ST 2110-10:2022 Section 8.7.`));
+      }
+    }
+    if (keys.indexOf('TSDELAY') >= 0) {
```

Comment:
> Not so sure what a "decimal positive integer" is. 

Not very clear but I think it means base-10 representation of the integer rather than e.g. hexadecimal!

Reply:
> This misses the test that TSDELAY must be positive. If the SDP has for example -100 then the error is not caught.

@jonathan-r-thorpe that seems like a good thing to add after the `isNaN` condition, e.g. `|| stream.TSDELAY <= 0` ?

Reply:
Agreed. Will update